### PR TITLE
Customers API

### PR DESCRIPTION
**Pull request by glendaesutanto**
- `POST` Create customer
https://developers.xendit.co/api-reference/#create-customer

- `GET` Get customer by reference ID
https://developers.xendit.co/api-reference/#get-customer-by-reference-id

### ps
The `qrcode` file is carried over because I run pre-commit that automatically reformat code styling, and it turns out some of the `qrcode` files were reformatted as well

### CODE REVIEW

File: customer/client.go [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+		c.Opt.SecretKey,
+		nil,
+		nil,
+		&response,
+	)
+	if err != nil {
+		return nil, err
+	}
+
+	// response := []CustomerResponse{tempResponse.toCustomerResponse()}
```
Comment (bxcodec) [inline customer/client.go:83]: please remove

File: customer.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package xendit
+
+type CustomerAddress struct {
```
Comment (bxcodec) [inline customer.go:3]: Can add a comment for Documentation in pkg.go.dev?

File: customer/client.go (hunk 1 above)
Comment (glendaesutanto) [reply customer/client.go:83]: my bad, will remove it

File: customer.go (hunk 2 above)
Comment (glendaesutanto) [reply customer.go:3]: I'm actually kinda confused about this. There is that kind of comment in other examples but I can not find `Customer` in pkg.go.dev. What should I do? Am I understanding it wrong?

Comment (bxcodec) [reply customer.go:3]: I think the comment will be reflected once we release the SDK version

Comment (bxcodec) [reply customer.go:3]: You can try to run `godoc` in your local to see if it's works. 
If you can see the docs in local by using `godoc` you should be able to see it later in pkg.go.dev  once we release new version

Comment (glendaesutanto) [reply customer.go:3]: I try to install `godoc` by typing this command `go get -v  golang.org/x/tools/cmd/godoc`. But after that when I try to run `godoc` it's said that command not found. Am I doing it wrong? @bxcodec

Comment (glendaesutanto) [reply customer.go:3]: Ah nvm, is this what you mean? @bxcodec 
image

Comment (glendaesutanto) [reply customer.go:3]: I've installed `godoc` and when I run `godoc -http=:6060`, it redirects me into this page but I can't find the documentation for the project.
image
I've also tried to change the `pkg.go.dev` here https://pkg.go.dev/github.com/xendit/xendit-go into `localhost:6060` (it becomes `localhost:6060/github.com/xendit/xendit-go`) but it didn't work as well. Am I doing it wrong? @bxcodec

Comment (glendaesutanto) [reply customer.go:3]: The issue has been resolved

**Review by bxcodec — APPROVED** [review]
LGTM

Please do a test before releasing the new tag version

**Pull request merged**